<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainMenu</class>
 <widget class="QMainWindow" name="mainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>mainMenu</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1920</width>
      <height>1080</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="main">
     <widget class="QPushButton" name="Main2QuitGame">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>861</x>
        <y>880</y>
        <width>200</width>
        <height>100</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Main2Level1">
      <property name="geometry">
       <rect>
        <x>861</x>
        <y>160</y>
        <width>200</width>
        <height>100</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>100</height>
       </size>
      </property>
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Main2Manual">
      <property name="geometry">
       <rect>
        <x>861</x>
        <y>640</y>
        <width>200</width>
        <height>100</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>100</height>
       </size>
      </property>
      <property name="text">
       <string>Manual</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Main2Garage">
      <property name="geometry">
       <rect>
        <x>861</x>
        <y>280</y>
        <width>200</width>
        <height>100</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>100</height>
       </size>
      </property>
      <property name="text">
       <string>Garage</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Main2Credits">
      <property name="geometry">
       <rect>
        <x>861</x>
        <y>760</y>
        <width>200</width>
        <height>100</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>100</height>
       </size>
      </property>
      <property name="text">
       <string>Credits</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Main2Highscore">
      <property name="geometry">
       <rect>
        <x>861</x>
        <y>400</y>
        <width>200</width>
        <height>100</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>100</height>
       </size>
      </property>
      <property name="text">
       <string>Highscore</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Main2Settings">
      <property name="geometry">
       <rect>
        <x>861</x>
        <y>520</y>
        <width>200</width>
        <height>100</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>100</height>
       </size>
      </property>
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
     <widget class="QLabel" name="mainBackground">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1920</width>
        <height>1080</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>mainBackground</zorder>
     <zorder>Main2QuitGame</zorder>
     <zorder>Main2Level1</zorder>
     <zorder>Main2Manual</zorder>
     <zorder>Main2Garage</zorder>
     <zorder>Main2Credits</zorder>
     <zorder>Main2Highscore</zorder>
     <zorder>Main2Settings</zorder>
    </widget>
    <widget class="QWidget" name="level1">
     <widget class="QPushButton" name="Level1_2Main">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>108</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>Main Menu</string>
      </property>
     </widget>
     <widget class="QLabel" name="level1Background">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1920</width>
        <height>1080</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="level1TrackPic">
      <property name="geometry">
       <rect>
        <x>647</x>
        <y>271</y>
        <width>120</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string> Track picture level 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="level1Trackname">
      <property name="geometry">
       <rect>
        <x>873</x>
        <y>151</y>
        <width>108</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Trackname Level 1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="playLevel1">
      <property name="geometry">
       <rect>
        <x>873</x>
        <y>535</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Level1_2Level3">
      <property name="geometry">
       <rect>
        <x>421</x>
        <y>390</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>left</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Level1_2Level2">
      <property name="geometry">
       <rect>
        <x>1325</x>
        <y>390</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>right</string>
      </property>
     </widget>
     <zorder>level1Background</zorder>
     <zorder>Level1_2Main</zorder>
     <zorder>level1TrackPic</zorder>
     <zorder>level1Trackname</zorder>
     <zorder>playLevel1</zorder>
     <zorder>Level1_2Level3</zorder>
     <zorder>Level1_2Level2</zorder>
    </widget>
    <widget class="QWidget" name="level2">
     <widget class="QPushButton" name="Level2_2Main">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>108</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>Main Menu</string>
      </property>
     </widget>
     <widget class="QLabel" name="level2Background">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1920</width>
        <height>1080</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>190</y>
        <width>1125</width>
        <height>411</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="2">
        <widget class="QLabel" name="level2Trackname">
         <property name="text">
          <string>Trackname Level 2</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="playLevel2">
         <property name="text">
          <string>Play</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QPushButton" name="Level2_2Level3">
         <property name="text">
          <string>right</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="Level2_2Level1">
         <property name="text">
          <string>left</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" rowspan="3" colspan="3">
        <widget class="QLabel" name="level2TrackPic">
         <property name="text">
          <string> Track picture level 2</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <zorder>level2Background</zorder>
     <zorder>Level2_2Main</zorder>
     <zorder>layoutWidget</zorder>
    </widget>
    <widget class="QWidget" name="level3">
     <widget class="QPushButton" name="Level3_2Main">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>108</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>Main Menu</string>
      </property>
     </widget>
     <widget class="QLabel" name="level3Background">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1920</width>
        <height>1080</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>180</y>
        <width>1125</width>
        <height>411</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="2">
        <widget class="QLabel" name="level3Trackname">
         <property name="text">
          <string>Trackname Level 3</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="playLevel3">
         <property name="text">
          <string>Play</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QPushButton" name="Level3_2Level1">
         <property name="text">
          <string>right</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="Level3_2Level2">
         <property name="text">
          <string>left</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" rowspan="3" colspan="3">
        <widget class="QLabel" name="level3TrackPic">
         <property name="text">
          <string> Track picture level 3</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <zorder>level3Background</zorder>
     <zorder>Level3_2Main</zorder>
     <zorder>layoutWidget_2</zorder>
    </widget>
    <widget class="QWidget" name="garage">
     <widget class="QPushButton" name="Garage2Main">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>108</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>Main Menu</string>
      </property>
     </widget>
     <widget class="QLabel" name="garageBackground">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1920</width>
        <height>1080</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="titel">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>130</y>
        <width>331</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Garage</string>
      </property>
     </widget>
     <widget class="QSlider" name="sliderAcceleration">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>330</y>
        <width>160</width>
        <height>18</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="acceleration">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>260</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Acceleration</string>
      </property>
     </widget>
     <widget class="QLabel" name="topSpeed">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>400</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Top Speed</string>
      </property>
     </widget>
     <widget class="QSlider" name="sliderTopspeed">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>450</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="sliderHandling">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>580</y>
        <width>160</width>
        <height>18</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="handling">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>510</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Handling</string>
      </property>
     </widget>
     <zorder>garageBackground</zorder>
     <zorder>Garage2Main</zorder>
     <zorder>titel</zorder>
     <zorder>sliderAcceleration</zorder>
     <zorder>acceleration</zorder>
     <zorder>topSpeed</zorder>
     <zorder>sliderTopspeed</zorder>
     <zorder>sliderHandling</zorder>
     <zorder>handling</zorder>
    </widget>
    <widget class="QWidget" name="highscore">
     <widget class="QPushButton" name="Highscore2Main">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>108</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>Main Menu</string>
      </property>
     </widget>
     <widget class="QLabel" name="highscoreBackground">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1920</width>
        <height>1080</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="highscoreLevel1">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>210</y>
        <width>501</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Highscore Level 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="highscoreLevel2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>480</y>
        <width>501</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Highscore Level 3</string>
      </property>
     </widget>
     <widget class="QLabel" name="highscoreLevel3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>350</y>
        <width>501</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Highscore Level 2</string>
      </property>
     </widget>
     <zorder>highscoreBackground</zorder>
     <zorder>Highscore2Main</zorder>
     <zorder>highscoreLevel1</zorder>
     <zorder>highscoreLevel2</zorder>
     <zorder>highscoreLevel3</zorder>
    </widget>
    <widget class="QWidget" name="settings">
     <widget class="QPushButton" name="Settings2Main">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>108</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>Main Menu</string>
      </property>
     </widget>
     <widget class="QLabel" name="settingsBackground">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1920</width>
        <height>1080</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="soundOn">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>350</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>On</string>
      </property>
     </widget>
     <widget class="QPushButton" name="soundOff">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>350</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Off</string>
      </property>
     </widget>
     <widget class="QLabel" name="soundLabel">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>350</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Sound</string>
      </property>
     </widget>
     <zorder>settingsBackground</zorder>
     <zorder>Settings2Main</zorder>
     <zorder>soundOn</zorder>
     <zorder>soundOff</zorder>
     <zorder>soundLabel</zorder>
    </widget>
    <widget class="QWidget" name="manual">
     <widget class="QPushButton" name="Manual2Main">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>108</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>Main Menu</string>
      </property>
     </widget>
     <widget class="QLabel" name="manualBackground">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1920</width>
        <height>1080</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="manualLabel">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>140</y>
        <width>431</width>
        <height>221</height>
       </rect>
      </property>
      <property name="text">
       <string>Manual - How to drive</string>
      </property>
     </widget>
     <zorder>manualBackground</zorder>
     <zorder>Manual2Main</zorder>
     <zorder>manualLabel</zorder>
    </widget>
    <widget class="QWidget" name="credits">
     <widget class="QPushButton" name="Credits2Main">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>108</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>Main Menu</string>
      </property>
     </widget>
     <widget class="QLabel" name="creditsBackground">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1920</width>
        <height>1080</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="creditsLabel">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>290</y>
        <width>221</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>Credits:</string>
      </property>
     </widget>
     <zorder>creditsBackground</zorder>
     <zorder>Credits2Main</zorder>
     <zorder>creditsLabel</zorder>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>108</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
     </font>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Comic Sans MS'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2';&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:36pt; font-weight:600;&quot;&gt;Island Racer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
